src/components/accordion/accordion.stories.mdx

```diff
@@ -177,6 +178,70 @@ import { Accordion, AccordionGroup } from "carbon-react/lib/components/accordion
   </Story>
 </Preview>
 
+### With validation icon
+The `ValidationIcon` can be displayed in the `Accordion` header by passing in a string via one of the `error`, 
+`warning` or `info` props. This message will be displayed as in the tooltip.
+
+<Preview>
+  <Story name="validation icon">
+    {() => {
+      const [errors, setErrors] = useState({ one: 'error', two: 'error' });
+      const [warnings, setWarnings] = useState({});
+      const [infos, setInfos] = useState({});
+      const handleChange = (id, type, setter, msg) => {
```

Comment:
```suggestion
      const handleChange = (id, setter, msg) => {
```

Reply:
I need the type param here so I can do `const update = type[id] ? undefined : msg;`